src/lib/address/address.ts

```diff
@@ -27,10 +29,10 @@ const CHAIN_ID_TO_URN_SCHEME: Record<ChainId, string> = {
   [ethChainId]: 'ethereum',
   [btcChainId]: 'bitcoin',
   [bchChainId]: 'bitcoincash',
-  [dogeChainId]: 'dogecoin',
+  [dogeChainId]: 'doge',
```

Comment:
I'm not across the reasoning for this change, but I tested a DOGE send and it still worked.

Reply:
We initially followed manually constructed examples in https://trezor.io/learn/a/uri-handlers-for-bitcoin-payments

<img width="934" alt="image" src="https://user-images.githubusercontent.com/17035424/236141830-dce704c6-917d-4d86-b826-4390e0c6aae8.png">

The reality is that apart from `bitcoin`, it's up to the implementooors to use the URN scheme they want. Trezor assumed `dogecoin` but Trust actually uses `doge`, which was causing some sneaky bug